cubi_tk/snappy/pull_processed_data.py

```diff
@@ -298,7 +299,7 @@ def pair_ipath_with_outdir(self, remote_files_dict, output_dir, assay_uuid, retr
         """Pair iRODS path with local output directory
 
         :param remote_files_dict: Dictionary with iRODS collection information. Key: file name as string (e.g.,
-        'P001-N1-DNA1-WES1'); Value: iRODS data (``IrodsDataObject``).
+        'P001-N1-DNA1-WES1'); Value: iRODS data (``DataObject``).
```

Comment:
I think the correct calls/type name here is iRODSDataObject (at least its imported under that name in check.py; inside irods it iirc called data_object)